exo-go/src/process_helpers/index.go

```diff
@@ -34,7 +34,9 @@ func RunAndLog(dir string, env []string, log func(string), commandWords ...strin
 	}
 	process := NewProcess(commandWords...)
 	process.SetDir(dir)
-	process.SetEnv(env)
+	if len(env) > 0 {
+		process.SetEnv(env)
+	}
```

Comment:
I suggest doing it in this PR. I don't think the current change is a good one. Should be a quick change to apply this and I will deal with merge conflicts in my PR since mine isn't passing yet anyway